src/components/menu/menu-popover.tsx

```diff
@@ -7,19 +7,18 @@ export const calculatePopoverPosition = (
   container: HTMLElement,
   popover: HTMLDivElement,
   setPopoverStyle: React.Dispatch<React.SetStateAction<React.CSSProperties>>,
-  additionalGap = 0,
+  yOffset = 0,
```

Comment:
**suggestion:** Andrei confirmed in Figma that the default can be `--space-1` which is 4px. Be good to get that locked in now. I'd also suggest moving the default value to the component, not this helper.

Reply:
Ok 👌

Reply:
but I think its not applicable for this case to put the default spacing in css, because it will always be overriden by the `calculatePopoverPosition`, even if that function disabled, `top: -4px` will be covering the button. I think I will add this to the function and add a comment for it

Reply:
Yeah, I understand `calculatePopoverPosition` will override any initial offset, my suggestion for moving the 4px default to the component is one of ownership. In my view, it is the component that owns the default, not `calculatePopoverPosition`.

Not sure where you're getting `-4px` from though. Maybe I'm missing something here, but the offset is an absolute value, whether it's positive or negative is up to `calculatePopoverPosition` to determine based on where it decides to place the popover, right?

Reply:
> Not sure where you're getting -4px from though.
apologize, that should be `4px` (var(--spacing-1) or 4px or 0.25rem)

> Maybe I'm missing something here, but the offset is an absolute value, whether it's positive or negative is up to calculatePopoverPosition to determine based on where it decides to place the popover, right?

yes, its absolute value of the parent (Menu) so if its default 4px it will be covering the trigger (since the trigger is tipically render before menu). the calculate fn is also responsible to position the mene above or below the trigger

Reply:
Ok, I think I got it now, I think I can move the default to the component props instead of inside helpers, last week I thought it should be on the css which is not possible to combine it with the helper 😆 .